script/preview.js

```diff
@@ -24,6 +24,17 @@ const COMMAND_LINE_OPTIONS_DEFINITIONS = [
   { name: 'protocol', type: String, defaultValue: 'http' },
   { name: 'destination', type: String, defaultValue: null },
   { name: 'content-directory', type: String, defaultValue: defaultContentDir },
+  {
+    name: 'drupal-address',
+    type: String,
+    defaultValue: process.env.DRUPAL_ADDRESS,
+  },
+  { name: 'drupal-user', type: String, defaultValue: process.env.DRUPAL_USER },
+  {
+    name: 'drupal-password',
+    type: String,
+    defaultValue: process.env.DRUPAL_PASSWORD,
+  },
```

Comment:
I can't decide if doing it this way was overkill, but centralizing these values here seemed consistent with how our site accepts build commands versus a floating `process.env.DRUPAL_USER` in our Drupals constants.